WifiART/src/structs/BssidState.java

```diff
@@ -0,0 +1,26 @@
+package structs;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BssidState {
+
+    private List<String> bssidList;
+
+    public BssidState() {
+        bssidList = new ArrayList<>();
+    }
+
+    public void addBssid(String bssid) {
+        bssidList.add(bssid);
+    }
+
+    public void deleteBssid(int index) {
```

Comment:
How does the caller knows the index to delete? 'index' is related to bssidList, which is an internal implementation.